lib/commons/aria/label.js

```diff
@@ -2,21 +2,18 @@
 /**
  * Gets the accessible ARIA label text of a given element
  * @see http://www.w3.org/WAI/PF/aria/roles#namecalculation
- * @param  {Object} node The virtual DOM node to test
- * @return {Mixed}      String of visible text, or `null` if no label is found
+ * @param  {Object} node	The DOM node to test
+ * @return {Mixed}      	String of visible text, or `null` if no label is found
  */
 aria.label = function (node) {
-	var ref, candidate;
-	if (node.actualNode instanceof Node === false) {
-		node = axe.utils.getNodeFromTree(axe._tree[0], node);
-	}
+	let ref, candidate;
 
-	if (node.actualNode.getAttribute('aria-labelledby')) {
+	if (node.getAttribute('aria-labelledby')) {
```

Comment:
Being able to use `node` instead of `node.actualNode` makes our API much easier to work with. 👍